src/vn/edu/rmit/kuri/output/Display.java

```diff
@@ -174,7 +174,14 @@ public static void chart(Summary summary, Metric metric) {
       valuePositionOnChart.add(position);
     }
 
-    spacing = (int) Math.floor((float) (horizontal - 1) / valueForDisplay.size());  // TODO: This is not accurate. Fix it.
+    //create spacing between data points on the chart
+    spacing = (int) Math.floor((float) (horizontal) / valueForDisplay.size());
+    /*check if the remaining space between the last data point and the horizontal edge is smaller
```

Comment:
I think horizontalEdge is kind of misleading, u can use something like spaceRemaining.

Reply:
remainingSpace adheres more to the naming conventions. I'll use it. Thanks!